Kickstart scraper

souravtecken opened the pull request:
Added kickstart.py - Scraper
1. Scraper for kickstart's practice round.
2. Creates kickstart.csv - containing Rank, UserName, Score of all participants

souravtecken committed:
Added kickstart.py
1. Scraper for kickstart's practice round.
2. Creates kickstart.csv - containing Rank, UserName, Score of all participants

souravtecken committed:
Merge branch 'master' of https://github.com/varunvora/alcoding

varunvora commented:
Can you add an option to make another output similar to the ones in /database/contest_ranks ? Just the list of usernames by their rank.

souravtecken committed:
Ran scraper for kickstart:
1. kickstart-pratice-2019.in generated in /scrapers

varunvora committed:
optimized scraper and PEP8 modifications for issue #5

souravtecken committed:
Ran scraper for codeJam 2018 qualifiers.
1. Scraper works for both codeJam and kickstart.
2. Created codejam-qualifiers-2018.in

varunvora committed:
removed scraped files
